tests/python_package_test/test_engine.py

```diff
@@ -1072,6 +1072,46 @@ def test_cvbooster():
     assert ret < 0.15
 
 
+def test_cvbooster_save_load(tmp_path):
+    X, y = load_breast_cancer(return_X_y=True)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.1, random_state=42)
+    params = {
+        'objective': 'binary',
+        'metric': 'binary_logloss',
+        'verbose': -1,
+    }
+    nfold = 3
+    lgb_train = lgb.Dataset(X_train, y_train)
+
+    def predict(cv_booster):
+        return np.array(cv_booster.predict(X_test))
+
+    cv_res = lgb.cv(params, lgb_train,
+                    num_boost_round=10,
+                    nfold=nfold,
+                    callbacks=[lgb.early_stopping(stopping_rounds=5)],
+                    return_cvbooster=True)
+    cvbooster = cv_res['cvbooster']
+
+    preds = predict(cvbooster)
+
+    model_path_txt = str(tmp_path / 'lgb.model')
+    model_path_pkl = str(tmp_path / 'lgb.pkl')
+
+    cvbooster.save_model(model_path_txt)
+    with open(model_path_pkl, 'wb') as f:
+        pickle.dump(cvbooster, f)
+
+    preds_from_file = predict(lgb.CVBooster(model_file=model_path_txt))
```

Comment:
```suggestion
    preds_from_txt_file = predict(lgb.CVBooster(model_file=model_path_txt))
```

Could you please rename this `preds_from_txt_file`? To make it clear how this is different from `preds_from_pkl` (since those are also produced based on a model stored in a file).